config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
+    namespace :api do
+      namespace :v1 do
+        resources :favorites
+        resources :computers
+        resources :users
```

Comment:
- [ ] Do you need all of the created routes? In my opinion it's better to specify which routes you really need, since you aren't using all of them.